protobuf/check_customization_applied.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package Trento.Checks.V1;
+option go_package = "/;events";
+
+import "google/protobuf/struct.proto";
+import "protobuf/check_custom_value.proto";
+
+message CheckCustomizationApplied {
+    string check_id = 1;
+    string group_id = 2;
+    repeated CheckCustomValue custom_values = 3;
+    map<string, google.protobuf.Value> metadata = 4;
```

Comment:
comment: I understand that you look for versatility here, what in reality we only have 2 things that we really want to send as far as I see. The target type and the user.  Do you see any other value that we really need to send? Sending the whole check looks unnecessary.
Sending specifically the expected fields could be a good idea, so the contract is strict and we don't open up things to send anything.
I would only send an open map if we have real doubts

Reply:
I'd be fine in sending only the target type, instead of the whole metadata. At the end at this stage it could be enough as we need it to query back the correct name of the target when adding a new entry in the activity log.

With regards to the user I believe we could add it in rabbit's message metadata so we do not pollute messages in the contracts.
I guess @CDimonaco had an opinion on this.